experiments/exp1/polynomial_critic.py

```diff
@@ -86,44 +79,52 @@ def setup_graph(self, state_input, action_input, net, linear):
         print("state_input {0}. action_input {1}".format(state_input, action_input))
         return q_value_output, net
 
+    def create_poly_q(self):
+        """ Initialize the polynomial critic network"""
+        # TODO: FIX TO REFLECT NEW CHANGES TO OBJECT.
+        # Create the variables.
+        linear = self.order <= 1
+        layer_size = self.state_dim + self.action_dim if not linear else 1
+        # TODO: ensure no conflict between dimension objects and ints

```

Comment:
pretty sure this todo is a leftover from when i was working on the polynomial critic. can be removed.